### PR TITLE
silution

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
solution*
[DEMO LINK](https://Zelinskyi-Serhii.github.io/react_tabs-with-router/)

### CODE REVIEW

File: src/TabItem/TabItem.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export const TabItem: FC<Props> = ({ tab, isSelected }) => (
+  <li
+    data-cy="Tab"
+    className={classNames({
+      'is-active': isSelected,
+    })}
+  >
+    <Link to={`../${tab.id}`}>
+      {tab.title}
+    </Link>
```
Comment (rashyd-hasratov) [inline src/TabItem/TabItem.tsx:20]: Let`s use destructuring for tab

**Review by rashyd-hasratov — CHANGES_REQUESTED** [review]
Good job! You just forgot to add README (DEMO LINK doesn't work) and also I left one more suggestion below

File: src/TabsPage/TabsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          })}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {activeTad ? (
+          activeTad.content
+        ) : (
+          'Please select a tab'
+        ) }
```
Comment (OleksiiBlezniuk) [inline src/TabsPage/TabsPage.tsx:37]:
```suggestion
        {activeTad
          ? (
            activeTad.content
          )
          : (
            'Please select a tab'
          ) }
```

File: src/TabsPage/TabsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import { Tab } from '../types/Tab';
+import { TabItem } from '../TabItem/TabItem';
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const TabsPage: FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+  const activeTad = tabs.find(tab => tab.id === tabId);
```
Comment (OleksiiBlezniuk) [inline src/TabsPage/TabsPage.tsx:12]:
```suggestion
  const activeTab = tabs.find(tab => tab.id === tabId);
```

File: src/TabsPage/TabsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import { useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+import { TabItem } from '../TabItem/TabItem';
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const TabsPage: FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
```
Comment (OleksiiBlezniuk) [inline src/TabsPage/TabsPage.tsx:11]: better to create type for params and pass it as generic here

**Review by OleksiiBlezniuk — APPROVED** [review]

**Pull request closed without merging**